jsapp/js/account/usage/usageProjectBreakdown.tsx

```diff
@@ -69,90 +34,63 @@ const ProjectBreakdown = () => {
         };
       });
 
-      setProjectData((prevData) => {return {...prevData, results: updatedResults};});
+      if (isComponentMounted) {
+        setProjectData({
+          ...data,
+          results: updatedResults,
+        });
+      }
     }
-  };
-
- const onFetchAssetsDataDone = (assetsData: any, updatedResults: ProjectResult[]) => {
-  if (isComponentMounted) {
-    const updatedData = updatedResults
-      .filter((projectResult) => {
-        const matchingAsset = assetsData.results.find(
-          (asset: ProjectViewAsset) => asset.uid === projectResult.uid
-        );
-
-        // Include the asset in the updatedData only if it is a survery
-        return matchingAsset?.asset_type === 'survey';
-      })
-      .map((projectResult) => {
-        const matchingAsset = assetsData.results.find(
-          (asset: ProjectViewAsset) => asset.uid === projectResult.uid
-        );
-
-        if (matchingAsset) {
-          return {
-            ...projectResult,
-            assets: [matchingAsset],
-            deployment_status: matchingAsset.deployment_status,
-          };
-        }
-        return projectResult;
-      });
-
-    setProjectData((prevData: Project) => {
-      return {
-        ...prevData,
-        results: updatedData as ProjectResult[],
-      };
-    });
-  }
-};
 
-  // This code will be updated to only have one api call when the asset_usage api is updated
-  // Right now, you may need to refresh or re-navigate to the page multiple times as the Loading Spinner can be shown indefinitely
-  useEffect(() => {
-    const usageUrl = `${ROOT_URL}/api/v2/asset_usage`;
-
-    const fetchData = $.ajax({
-      dataType: 'json',
-      method: 'GET',
-      url: usageUrl,
-    })
-      .done(onFetchAssetsDone)
-      .then(() => {
-        const assetsUrl = `${ROOT_URL}/api/v2/assets`;
-
-        return $.ajax({
-          dataType: 'json',
-          method: 'GET',
-          url: assetsUrl,
-        })
-          .done((assetsData) =>
-            onFetchAssetsDataDone(assetsData, projectData.results)
-          )
-          .fail(onAnyFail);
-      })
-      .fail(onAnyFail);
+    fetchData();
 
     return () => {
       setIsComponentMounted(false);
     };
+  }, [isComponentMounted]);
```

Comment:
When I open the tab, the links to the projects work:
![image](https://github.com/kobotoolbox/kpi/assets/7819986/604b149f-8583-48c2-9259-e3e4941549a6)

But after I switch pages, it looks like the asset UID is missing:
![image](https://github.com/kobotoolbox/kpi/assets/7819986/48fa9906-138e-4de5-a880-15adae7b28cf)

Looks like that's happening because this `useEffect()` doesn't run when the page changes.

Here's what I'd suggest: what if `getAssetUsageForOrganization()` took a page number (`currentPage`), instead of a URL, and you made this `useEffect()` depend on `currentPage`? Then you don't have to do any fetching in `handleClick()` - just increment/decrement the page number. You'll need to do something similar when adding sorting, so this should be good practice 👍 

This would also allow us to, in the future, take a page number in the URL (e.g. `/#/account/usage?p=2`) and jump